cime_config/testdefs/testlist_clm.xml

```diff
@@ -2411,8 +2313,7 @@
   </test>
   <test name="SMS_Lm1" grid="f19_g17" compset="I1850Clm50BgcCropCmip6waccm" testmods="clm/basic">
     <machines>
-      <machine name="cheyenne" compiler="intel" category="aux_clm"/>
-      <machine name="cheyenne" compiler="intel" category="prebeta"/>
+      <machine name="cheyenne" compiler="intel" category="ctsm_sci"/>
```

Comment:
- [ ] Add a note here about why this test is in the ctsm_sci test list.

Reply:
I know I just said that I'd do this, but I realized that there already is a comment on this test (you need to expand the diff below to see it: "The main point of this test is simply to make sure that the CMIP6WACCMDECK modifier works for 2-degree since that resolution turns off Carbon isotopes"). I'm not sure what else to say besides that.

Reply:
OK, I looked at the file to see what this looks like. My point is that the other ctsm_sci tests are all for running the science supported compsets/resolutions. This isn't listed as a science_supported compset or resolution so I think it warrants saying why we have this in ctsm_sci. So I suggest adding something like this...

This isn't a science supported compset, but it's critical to run this test at 2-degree resolution which is higher than our standard testing.